code/lib/cli/src/add.test.ts

```diff
@@ -104,7 +104,18 @@ describe('add', () => {
 describe('add (extra)', () => {
   test('not warning when installing the correct version of storybook', async () => {
     await add(
-      'storybook/addon-docs',
+      '@storybook/addon-docs',
+      { packageManager: 'npm', skipPostinstall: true },
+      MockedConsole
+    );
+
+    expect(MockedConsole.warn).not.toHaveBeenCalledWith(
+      expect.stringContaining(`is not the same as the version of Storybook you are using.`)
+    );
+  });
+  test('not warning when installing unrelated package', async () => {
+    await add(
+      '@storybook/addon-docs',
```

Comment:
This test does the same as the one in line 105. I guess you want to use a non-core addon in line 118.